src/app/modules/main/communities/tokens/module.nim

```diff
@@ -84,32 +85,46 @@ proc authenticate(self: Module) =
   else:
     self.controller.authenticateUser()
 
-proc getTokenAndAmountList(self: Module, communityId: string, collectiblesJsonString: string): seq[CommunityTokenAndAmount] =
+# for collectibles conversion is: "1" -> Uint256(1)
+# for assets amount is converted to basic units (wei-like): "1.5" -> Uint256(1500000000000000000)
+proc convertAmountByTokenType(self: Module, tokenType: TokenType, amount: float64): Uint256 =
+  const decimals = 18
+  case tokenType
+    of TokenType.ERC721:
+      return stint.parse($amount.int, Uint256)
+    of TokenType.ERC20:
+      return conversion.eth2Wei(amount, decimals)
+    else:
+      error "Converting amount - unknown token type", tokenType=tokenType
+
+proc getTokenAndAmountList(self: Module, communityId: string, tokensJsonString: string): seq[CommunityTokenAndAmount] =
```

Comment:
Whenever we take json as a string it would be nice to have short description what's the expected structure of that json :)